pkg/webhook/seed/validation.go

```diff
@@ -190,6 +193,12 @@ func (v *validator) validate(ctx context.Context, obj runtime.Object, isDelete b
 		if _, exists := subject.Spec.EtcdBackupRestore.Destinations[subject.Spec.EtcdBackupRestore.DefaultDestination]; !exists {
 			return fmt.Errorf("invalid etcd backup configuration: default destination %q does not exist", subject.Spec.EtcdBackupRestore.DefaultDestination)
 		}
+
+		etcdBackupSecret := corev1.Secret{}
+		if err := seedClient.Get(ctx, types.NamespacedName{Name: resources.EtcdRestoreS3CredentialsSecret,
```

Comment:
we need to check if the credentials of each destination exist, not `resources.EtcdRestoreS3CredentialsSecret`.

`resources.EtcdRestoreS3CredentialsSecret` is a leftover of old code, ill remove it in a separate PR.

Reply:
Done